Tests/GISToolsTests/GeoJson/FeatureTests.swift

```diff
@@ -40,15 +41,57 @@ final class FeatureTests: XCTestCase {
         XCTAssertEqual(feature["prop0"], "value0")
         XCTAssertEqual(feature.foreignMember(for: "other"), "something else")
         XCTAssertEqual(feature[foreignMember: "other"], "something else")
+        XCTAssertEqual(feature.id, .string("abcd.1234"))
     }
 
-    func testCreateJson() {
-        // TODO:
+    private let featureJsonWithIntId = """
+    {
+       "type": "Feature",
+       "geometry": {
+           "type": "Polygon",
+           "coordinates": [
+               [
+                   [100.0, 0.0],
+                   [101.0, 0.0],
+                   [101.0, 1.0],
+                   [100.0, 1.0],
+                   [100.0, 0.0]
+               ]
+           ]
+       },
+       "properties": {
+           "prop0": "value0",
+           "prop1": {
+               "this": "that"
+           }
+       },
+       "other": "something else",
+       "id": 1234
+    }
+    """
+
+    func testLoadJsonWithIntId() throws {
+        guard let feature = Feature(jsonString: featureJsonWithIntId) else {
+            throw "feature is nil"
+        }
+
+        XCTAssertEqual(feature.id, .int(1234))
+    }
+
+
+    func testCreateJson() throws {
+        let feature = Feature(Point(.zero))
```

Comment:
I would propose to add an `id` here and check that it's of the proper type in the result JSON, like

```
        let feature = Feature(Point(.zero), id: .int(5))
        ...
        XCTAssertEqual(json["id"] as? Int, 5)
```